src/CECILL-1.0.xml

```diff
@@ -0,0 +1,468 @@
+      <p>Contributeur: désigne le Licencié auteur d’au moins une Contribution.</p>
+      <p>Concédant: désigne le Titulaire ou toute personne physique ou morale distribuant le Logiciel sous le
+         Contrat.</p>
+      <p>Contributions: désigne l’ensemble des modifications, corrections, traductions, adaptations et/ou
+         nouvelles fonctionnalités intégrées dans le Logiciel par tout</p>
+      <p>Contributeur, ainsi que les Modules Statiques.</p>
+      <p>Module: désigne un ensemble de fichiers sources y compris leur documentation qui, une fois compilé sous
+         forme exécutable, permet de réaliser des fonctionnalités ou 
+        <br/>services supplémentaires à ceux fournis par le Logiciel. 
+      </p>
+      <p>Module Dynamique: désigne tout Module, créé par le Contributeur, indépendant du Logiciel, tel que ce
+         Module et le Logiciel sont sous forme de deux exécutables indépendants qui s’exécutent dans un espace
+         d’adressage indépendant, l’un appelant l’autre au moment de leur exécution.</p>
+      <p>Module Statique: désigne tout Module créé par le Contributeur et lié au Logiciel par un lien statique
+         rendant leur code objet dépendant l'un de l'autre. Ce Module et le Logiciel auquel il est lié, sont
+         regroupés en un seul exécutable.</p>
+      <p>Parties: désigne collectivement le Licencié et le Concédant.</p>
+      <p>Ces termes s’entendent au singulier comme au pluriel.</p>
+      <p>Article 2 - OBJET</p>
+      <p>Le Contrat a pour objet la concession par le Concédant au Licencié d’une Licence non exclusive,
+         transférable et mondiale du Logiciel telle que définie ci-après à l'article 5 pour toute la durée de
+         protection des droits portant sur ce Logiciel.</p>
+      <p>Article 3 - ACCEPTATION</p>
+      <list>
+        <li>
+          <b>3.1.</b>
+          <p>L’acceptation par le Licencié des termes du Contrat est réputée acquise du fait du premier des
+             faits suivants:</p>
+        </li>
+        <list>
+          <li>
+            <b>•</b>
+            <p>(i) le chargement du Logiciel par tout moyen notamment par téléchargement à partir d’un
+               serveur distant ou par chargement à partir d’un support physique;</p>
+          </li>
+          <li>
+            <b>•</b>
+            <p>(ii) le premier exercice par le Licencié de l’un quelconque des droits concédés par le Contrat.</p>
+          </li>
+        </list>
+        <li>
+          <b>3.2.</b>
+          <p>Un exemplaire du Contrat, contenant notamment un avertissement relatif aux spécificités du
+             Logiciel, à la restriction de garantie et à la limitation à un usage par des utilisateurs
+             expérimentés a été mis à disposition du Licencié préalablement à son acceptation telle que
+             définie à l'article 3.1 ci dessus et le Licencié reconnaît en avoir pris connaissances.</p>
+          <p>Article 4 - ENTREE EN VIGUEUR ET DUREE</p>
```

Comment:
Article 4 at wrong indentation, and similar problems with list tags throughout document. Pull request submitted to fix list tags.